### PR TITLE
LTD-5267-template

### PR DESCRIPTION
**Pull request by seijihg**
https://uktrade.atlassian.net/browse/LTD-5267

### CODE REVIEW

File: assets/css/siel.css [hunk 1]
```diff
@@ -3,12 +3,13 @@
   src: url("../fonts/Helvetica.ttf") format("truetype");
 }
 .document * {
-  font-size: 12px;
+  font-size: 10px;
   font-family: Helvetica;
   word-wrap: break-word;
+  word-break: break-word;
```
Comment (hnryjmes) [inline assets/css/siel.css:9]: MDN docs say that `break-word` is deprecated, do you know if that matters to us? is there an alternative?

File: api/letter_templates/templates/letter_templates/siel.html [hunk 2]
```diff
@@ -264,7 +264,7 @@ <h1>Standard Individual Export Licence</h1>
       <tr>
         <td class="border-black" colspan="6">
           <span class="cell__heading">2b. Applicant reference</span>
-          <span class="cell__uppercase">{{ details.user_reference }}</span>
+          <span class="cell__uppercase width-text">{{ details.user_reference }}</span>
```
Comment (hnryjmes) [inline api/letter_templates/templates/letter_templates/siel.html:267]: it's interesting to me that we need this addition when we show the user reference here, but not in the other places in the document where we use
`<span class="cell__uppercase">{{ details.user_reference }}</span>`
and this doesn't seem to cause a problem (at least, looking at the example pdf attached to the ticket)

should we make these usages consistent? do we need to?

Comment (seijihg) [reply api/letter_templates/templates/letter_templates/siel.html:267]: other places have a lot more spaces only here we have a small cell; makes the issue with long reference. this limit the characters and wrap it below.

File: assets/css/siel.css [hunk 3]
```diff
@@ -3,12 +3,12 @@
...
 }
 .document * {
-  font-size: 12px;
+  font-size: 11px;
   font-family: Helvetica;
   word-wrap: break-word;
 }
 .document h1 {
-  font-size: 24px;
+  font-size: 20px;
```
Comment (hnryjmes) [inline assets/css/siel.css:11]: can you confirm with someone from UCD that we're ok to make these changes?

**Review by hnryjmes — COMMENTED** [review]
have tested the PR on dev2 and this does seem to fix the issue. however, I have made a comment about font size changes that I would prefer answered before approving

**Review by hnryjmes — APPROVED** [review]
approved as Jonathan (UCD) confirmed the changes were acceptable

File: assets/css/siel.css [hunk 4]
```diff
@@ -49,6 +49,9 @@
   font-style: italic;
   margin-bottom: 5px;
 }
+.document .max-width {
+  max-width: 140px;
```
Comment (saruniitr) [inline assets/css/siel.css:53]: if the reference is wider than this then it wraps?

**Review by saruniitr — APPROVED** [review]

**Pull request merged**